elastic/apm-server
# .ci: only publish package to storage v2

## Motivation/summary

Package storage is dead, long live package storage.

## Checklist

~- [ ] Update [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/main/CHANGELOG.asciidoc)~
~- [ ] Update [package changelog.yml](https://github.com/elastic/apm-server/blob/main/apmpackage/apm/changelog.yml) (only if changes to `apmpackage` have been made)~
~- [ ] Documentation has been updated~

## How to test these changes

Ensure preview packages are still being uploaded by querying `https://epr.elastic.co/search?package=apm&prerelease=true`, and checking the timestamp of the latest package is fresh.

## Related issues

None
